share/models/sources.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+from django.db import models
+
+from share.models.ingest import SourceConfig
+from share.models.ingest import NaturalKeyManager
+
+logger = logging.getLogger(__name__)
+__all__ = ('SourceStat',)
+
+
+class SourceStat(models.Model):
```

Comment:
Could you add a `grade` or `ok` field, so we can filter for sources that are having issues?